docs/standaard/documenten/index.md

```diff
@@ -106,6 +106,7 @@ De [releasenotes](./release_notes.md) van de versies staan beschreven op deze [p
 
 Versie   | Release datum | API specificatie
 -------- | ------------- | ----------------
+1.5.0    | xx-03-2024    | [ReDoc][documenten-1.5.0-redoc], [Swagger][documenten-1.5.0-swagger]
```

Comment:
Links naar Redoc en Swagger werken niet

Reply:
...bij gebrek aan definitieve links ;)

Reply:
Ja, maar je kunt er wel voor zorgen dat de links wel werken binnen deze branch. Dan hoef je bij de switch alleen maar de de branch-naam in de link te wijzigingen naar master.

Reply:
Pagina's aangemaakt met link naar https://raw.githubusercontent.com/VNG-Realisatie/gemma-zaken/9ebfeebb857b7f4f338aee95872eb6c5aee3e61a/api-specificatie/drc/1.5.x/1.5.0/openapi.yaml.

Reply:
In een separate PR, ondertussen opgelost. Dit had handiger gekund, maar we misten jouw ervaring ;)